dev_guide/builds.adoc

```diff
@@ -385,6 +384,31 @@ sourceStrategy:
 You can also manage environment variables defined in the `*BuildConfig*` with the
 xref:../dev_guide/environment_variables.adoc#overview[`oc set env`] command.
 
+[[adding-secrets-to-source-strategy-build-configs]]
+=== Adding Secrets to Source Strategy Build Configurations
+
+Add a secret to your build configuration so that it can access a private
+repository.
+
+. Create a new {product-title} project.
```

Comment:
I thought that all the documented procedures are starting from scratch, meaning by creating a project and then the rest (creating secret, adding it to BC, etc.). Either way (1. creating project for each process that you are documenting. 2.Document all common processes in single project) makes sense. It really just depends how you are doing it in the rest of the docs so it aligns with them.